lib/styles/colors.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+Map<String, Color> colors = {
+  "background": Color(0xff444444), // #444
+  "navbar": Color(0xff232323), // #232323
+  "text": Color(0xffe2e2e2), // #e2e2e2
+  "button": Color(0xff636363), // #636363
```

Comment:
Dałbym inny kolor guzika niż szary. To byłby pewnie główny motyw aplikacji. Możliwe, że navbar też dobrze by wyglądał w tym kolorze. Może któraś z tych palet byłaby lepsza: https://colorhunt.co/palettes/dark

Reply:
Proponuję zostawienie tych kolorów jako placeholdery i zaplanować spotkanie, na którym ustalona zostanie ostateczna paleta kolorów.

Reply:
Do wybrania lepszego koloru zmienię na taki kolor.
```suggestion
  "button": Color(0xfffd7014), // #fd7014
```